keyboards/handwired/twadlee/tp69/tp69.c

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2020 Tracy Wadleigh
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "tp69.h"
+#include "ps2.h"
+
+uint8_t ps2_error = PS2_ERR_NONE;
+
+void    ps2_host_init(void) {}
+uint8_t ps2_host_send(uint8_t data) {return 0;}
+uint8_t ps2_host_recv_response(void) {return 0;}
+uint8_t ps2_host_recv(void) {return 0;}
+void    ps2_host_set_led(uint8_t usb_led) {}
```

Comment:
Given this is entirely stubs, does `PS2_MOUSE_ENABLE` add any value at all?

Reply:
Not yet :-) After I opened this PR I spent the rest of the weekend determining the right implementation appropriate for my Teensy LC. I think I made substantial headway on that. I intended for this PR to be focused on the keyboard only. I probably should have branched again before adding these stubs.

I'll back that last commit out of this branch.

Reply:
@zvecr stubs removed on this branch.

Reply:
As a FYI, #6490 does a lot of the refactor required to get it to build on ChibiOS.